Dockerfile

```diff
@@ -1,15 +1,10 @@
 FROM ubuntu
 
 RUN apt update && apt install -y git make unzip curl g++ python3 pip wget
-COPY /build/requirements.txt ./
-RUN pip3 install -r requirements.txt
+RUN pip3 install Pillow Wave
```

Comment:
Better to use "requireemnts.txt"? since all python package handling is isolated from Docker file but only in "requirements.txt"?

Reply:
The intention of these modifications was to make the smallest possible Docker image to run the Hello World test and show the test results on the terminal. So these are for demonstration purposes only; not intended to be in the final version of the Dockerfile.